test/helpers/kubectl.ts

```diff
@@ -67,6 +67,12 @@ export async function createDeploymentFromImage(name: string, image: string, nam
   console.log(`Done Letting Kubernetes decide how to manage image ${image} with name ${name}`);
 }
 
+export async function deleteResource(kind: string, name: string, namespace: string) {
```

Comment:
shouldn't all other delete* functions use this one?

Reply:
Yes, but part of a general improvement, didn't want to touch too much code in this PR, only what is relevant

Reply:
capture somehow in jira please